README.md

```diff
@@ -0,0 +1,54 @@
+# Fluentd Aggregator Docker Image
+
+[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/fluent/fluentd-aggregator?sort=semver)](https://hub.docker.com/r/fluent/fluentd-aggregator)
+![linux](https://img.shields.io/badge/os-linux-brightgreen)
+![amd64](https://img.shields.io/badge/arch-amd64-brightgreen)
+![arm64](https://img.shields.io/badge/arch-arm64-brightgreen)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+A [Fluentd](https://www.fluentd.org/) [OCI](https://opencontainers.org/) image to be used for log aggregation and based on the official [Fluentd Docker image](https://github.com/fluent/fluentd-docker-image) rebuilt as a multi-arch `linux/amd64` & `linux/arm64` image (until the official image support multi-arch).
```

Comment:
I think you need to indicate the Alpine base here as well.

Reply:
It is the default Fluentd image?